src/Build/BackEnd/Node/MSBuildClientExitResult.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Microsoft.Build.Experimental.Client
+{
+    /// <summary>
+    /// Enumeration of the various ways in which the MSBuildClient execution can exit.
+    /// </summary>
+    public class MSBuildClientExitResult
```

Comment:
Consider to make this type `sealed`.

Reply:
We will make this experimental in the next PR.